modules/about-certificate-management.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-a-custom-pki.adoc
+
+[id="about-certificate-management_{context}"]
+= About certificate management
```

Comment:
Just a question on whether this title is actually descriptive of what this section is discussing, or if there's something more specific it could say? It seems a little broad as is.

Reply:
@bergerhoffer what do you think about *Certificate injection using Operators* instead?

Reply:
Possibly. It does seem informational so I don't mind the About part, though usually I"m used to seeing the concept modules like this more at the top of an assembly as opposed to at the end.

I'm not positive, but I'm wondering if it would make more sense coupled with the information you have at the beginning of the assembly? Just something to think about, I'm fine with however you think it's best presented.